dev/WindowsAppRuntime_MSIXInstallFromPath/main.cpp

```diff
@@ -118,18 +118,18 @@ class PackageId
 void Help()
 {
     wprintf(L"WindowsAppRuntime_MSIXInstallFromPath [options] <path>\n"
-            L"options:\n"
-            L"  -?, --help = Display help\n"
-            L"          -- = End of options\n"
-            L"where:\n"
-            L"  path = Path where Windows App SDK's MSIX packages can be found\n"
-            L"           => Microsoft.WindowsAppRuntime.*.msix\n"
-            L"           => Microsoft.WindowsAppRuntime.DDLM.*.msix\n"
-            L"           => Microsoft.WindowsAppRuntime.Main.*.msix\n"
-            L"           => Microsoft.WindowsAppRuntime.Singleton.*.msix\n");
+        L"options:\n"
+        L"  -f, --force = Force shutdown the target applications if they are in use, to install the MSIX packages\n"
```

Comment:
Do we log anything that we're doing work with the Forced option? e.g. "Installing Package [Options=Force]: OK" or something? I don't see it but perhaps I'm just overlooking it

If not, we should. Here, or a follow up PR.

Reply:
Yes, we do already log Options in Install activity start event which captures Force or not.